src/javascript/JContent/actions/publishAction.jsx

```diff
@@ -70,7 +70,8 @@ export const PublishActionComponent = ({context, render: Render, loading: Loadin
         language: res.node.site ? _.escape(uppercaseFirst(getLanguageLabel(res.node.site.languages, language).displayName)) : null
     } : {
         displayName: t('jcontent:label.contentManager.selection.itemsSelected', {count: context.paths.length}),
-        language: res.nodes[0].site ? _.escape(uppercaseFirst(getLanguageLabel(res.nodes[0].site.languages, language).displayName)) : null
+        language: res.nodes[0].site ? _.escape(uppercaseFirst(getLanguageLabel(res.nodes[0].site.languages, language).displayName)) : null,
+        menuDisplayName: t('jcontent:label.contentManager.selection.items', {count: context.paths.length})
```

Comment:
name of the parameter is not very clear ?

Reply:
was not very inspired, it's the label of the publish action when there is multiple selection, anyway I'll modify it